### ISSUE TITLE
2016.3.0: Failed to load grains defined in grain file disks.disks (FreeBSD)

### ISSUE THREAD

**Issue by silenius** [saltstack/salt#33721]
### Description of Issue/Question

2016-06-02 22:58:10,033 [salt.loader                                                ][CRITICAL][98293] Failed to load grains defined in grain file disks.disks in function <function disks at 0x80d00d230>

### Setup

freebsd 10.3, salt 2016.0.3

### Steps to Reproduce Issue

sudo service salt_master start and a tail -f /var/log/salt/master:

2016-06-02 22:59:11,285 [salt.loader                                                ][CRITICAL][98293] Failed to load grains defined in grain file disks.disks in function <function disks at 0x80d00d1b8>, error:
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/salt/loader.py", line 724, in grains
    ret = fun()
  File "/usr/local/lib/python2.7/site-packages/salt/grains/disks.py", line 31, in disks
    return _freebsd_geom()
  File "/usr/local/lib/python2.7/site-packages/salt/grains/disks.py", line 107, in _freebsd_geom
    parse_geom_attribs(device)
  File "/usr/local/lib/python2.7/site-packages/salt/grains/disks.py", line 94, in parse_geom_attribs
    search.group(1))
  File "/usr/local/lib/python2.7/site-packages/salt/grains/disks.py", line 69, in _datavalue
    return int(data)
ValueError: invalid literal for int() with base 10: 'unknown'

### Versions Report

mage@mordor:/home/mage % salt --versions-report
Salt Version:
           Salt: 2016.3.0
 
Dependency Versions:
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: 2.5.0
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: 1.5.0
         Jinja2: 2.8
        libgit2: Not Installed
        libnacl: 1.4.4
       M2Crypto: Not Installed
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.4.7
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: 2.6.1
         pygit2: Not Installed
         Python: 2.7.11 (default, Apr 29 2016, 08:07:12)
   python-gnupg: Not Installed
         PyYAML: 3.11
          PyZMQ: 15.2.0
           RAET: 0.6.5
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 4.3
            ZMQ: 4.1.4
 
System Versions:
           dist:   
        machine: amd64
        release: 10.3-RELEASE
         system: FreeBSD
        version: Not Installed

**Comment by silenius** [comment]
this is with (may be used for a test case?):

```
mage@mordor:/home/mage % geom disk list
Geom name: ada0
Providers:
1. Name: ada0
   Mediasize: 2000398934016 (1.8T)
   Sectorsize: 512
   Mode: r6w6e12
   descr: WDC WD2001FASS-00W2B0
   lunid: 50014ee057e036d0
   ident: WD-WMAY00742604
   rotationrate: unknown
   fwsectors: 63
   fwheads: 16

Geom name: ada1
Providers:
1. Name: ada1
   Mediasize: 2000398934016 (1.8T)
   Sectorsize: 512
   Mode: r6w6e12
   descr: WDC WD2001FASS-00W2B0
   lunid: 50014ee0028edee1
   ident: WD-WMAY00637358
   rotationrate: unknown
   fwsectors: 63
   fwheads: 16
```

**Comment by silenius** [comment]
mmh sorry, apparently it has been fixed in the upstream/2016.3 branch
